test/main.sh

```diff
@@ -74,7 +74,7 @@ cleanup() {
     fi
     echo "==> Test result: ${TEST_RESULT}"
 
-    # shellcheck disable=SC2086
+    # shellcheck disable=SC2086,SC1117
     printf "To poke around, use:\n LXD_DIR=%s LXD_CONF=%s sudo -E %s/bin/lxc COMMAND\n" "${LXD_DIR}" "${LXD_CONF}" ${GOPATH:-}
```

Comment:
"\n" should just be replaced with "\\n" here which should get rid of the warning